app/res/layout/first_start_screen.xml

```diff
@@ -44,7 +44,7 @@
                <LinearLayout
                    android:id="@+id/message_panel"
                    android:layout_width="fill_parent"
-                   android:layout_height="wrap_content"
+                   android:layout_height="match_parent"
```

Comment:
Honestly don't remember if there's a reason for this change... I think it was just for testing. It'd be impossible to even have markup at this point since the app isn't installed. The parent also has wrap_content for its height so it probably just overrides this anyways. rolling back.